src/openms/source/FILTERING/DATAREDUCTION/FeatureFindingMetabo.cpp

```diff
@@ -147,6 +150,26 @@ namespace OpenMS
     return tmp_hulls;
   }
 
+  std::vector< OpenMS::MSChromatogram <> > FeatureHypothesis::getChromatograms(UInt64 feature_id) const
+  {
+    std::vector< OpenMS::MSChromatogram <> > tmp_chromatograms;
+    for (Size mt_idx = 0; mt_idx < iso_pattern_.size(); ++mt_idx)
+    {
+      OpenMS::MSChromatogram<> chromatogram;
```

Comment:
, I suggest to add some meta-information similar to what we do here: https://github.com/OpenMS/OpenMS/blob/develop/src/openms/include/OpenMS/ANALYSIS/OPENSWATH/ChromatogramExtractor.h#L252

e.g. add a precursor, set mz, set native id etc:

```
chromatogram.setNativeID(); // I suggest to match this to the feature id!
chromatogram.setChromatogramType(ChromatogramSettings::BASEPEAK_CHROMATOGRAM);
Precursor prec;
prec.setMZ();
chromatogram.setPrecursor(prec);

// Set the rest of the meta-data
chromatogram.setInstrumentSettings(settings.getInstrumentSettings());
chromatogram.setAcquisitionInfo(settings.getAcquisitionInfo());
chromatogram.setSourceFile(settings.getSourceFile());


for (Size j = 0; j < settings.getDataProcessing().size(); ++j)
{
  settings.getDataProcessing()[j]->setMetaValue("performed_on_spectra", "true");
  chromatogram.getDataProcessing().push_back(settings.getDataProcessing()[j]);
}
```